lib/processStaticFiles.js

```diff
@@ -13,7 +13,17 @@ function getEnvVariable(name) {
 
 const upload = () => {
   try {
+    if (fs.readdirSync(process.env.ENV_DIR).includes("IS_REVIEW_APP")) {
+      throw 'Review app detected'
+    }
+  } catch (error) {
+    console.error('Static Uploader is not configured for this deploy');
+    console.error(error);
+    console.error('Exiting without error');
+    process.exit(0);
+  }
 
+  try {
```

Comment:
A couple of things here:

1. If we're throwing an error, then immediately catching it and cutting off the rest of the execution, seems like it can be a conditional
2. Probably would be a good idea to generalize the name of the environment variable. Maybe something like `DISABLE_ASSET_UPLOAD`.
3. Seems like we should check the value of the environment variable. If I understand correctly, we're just checking for its existence currently? I can see that causing confusion, if someone sets that to false, thinking they've disabled something


```suggestion
    if (process.env.DISABLE_ASSET_UPLOAD === "true)) {
       console.error('Static Uploader is not configured for this deploy');
       console.error('Exiting without error');
       process.exit(0);
    }
```

Reply:
Thanks for this, @jjacobskind ! Re (1) and (2), good idea! I'll make those changes.

Re (3), it's a little complicated, since Heroku does not (despite what its docs say) store config vars directly in `process.env`. Rather, we have to read them (using `fs`) from the `process.env.ENV_DIR` directory. However, `fs.readFileSync` errors out (rather than returning false or undefined) if the file corresponding to the env var is not found. So my thought was to just check to see if it exists instead.

Maybe we can do something like this

```javascript
try {
  const shouldDisableAssetUpload = getEnvVar("DISABLE_ASSET_UPLOAD")
  if (shouldDisableAssetUpload) {
    throw 'Asset upload disabled'
  }
} catch (error) {
       console.error('Static Uploader is disabled for this deploy');
       console.error('Exiting without error');
       process.exit(0);
}
```
This would require adding the `DISABLE_ASSET_UPLOAD` to both staging and prod, but it would allow us to pass a `false` value

Reply:
Better yet, let's check to see if it exists and then read it! New commit incoming